config/aeotec/zw175.xml

```diff
@@ -146,10 +147,10 @@
         <Value genre="config" index="101" label="Configure which meter reading will be periodically report via Lifeline" type="int" units="" value="15">
             <Help>
                 Defines the type of report sent for reporting group 1.
```

Comment:
Note for reviewer: the description of the bits (L156-L160 in the base) below appear correct to me, it's just the integer representation that appear wrong.